react-nginx/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# 1. For build React app
+FROM node:10 AS builder
```

Comment:
NodeJS 10 has reached its end of life few weeks ago, can you change for an `node:lts` image or a `16:xxx` image ?